models/RwdImage.php

```diff
@@ -78,6 +78,32 @@ public function __construct( $attachment ) {
 	}
 
 	/**
+	 * Check if mime type SVG
+	 *
+	 * @return boolean
+	 */
+	public function is_svg_image() {
+		if ( preg_match( '/.svg/',  get_post_mime_type( $this->attachment->ID ) ) ) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+
+	/**
+	 * Check if mime type SVG
+	 *
+	 * @param int $attachment_id Attachment ID image.
+	 *
+	 * @return string
+	 */
+	public function get_svg_image( $attachment_id ) {
+		$image = get_attached_file( $attachment_id );
+		$svg = file_get_contents( $image );
+		return $svg;
```

Comment:
don't get what you do here.

The whole code should return correct <img> or <picture> or background css. You just get SVG content and print it???